src/content/en/updates/2018/06/ar-for-the-web.md

```diff
@@ -0,0 +1,277 @@
+
+{# wf_updated_on: 2018-06-15 #}
+{# wf_published_on: 2018-06-14 #}
+{# wf_tags: immersive-web,webar #}
+{# wf_blink_components: Blink>WebVR #}
+{# wf_featured_image: /web/updates/images/2018/06/sunflowers.png #}
+{# wf_featured_snippet: Augmented reality allows placement and tracking of virtual objects in a real-world view. #}
+
+# Augmented reality for the web {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In Chrome 67,
+[we announced](/web/updates/2018/05/welcome-to-immersive)
+the WebXR Device API for both augmented reality (AR) and virtual reality (VR),
+though only the VR features were enabled. VR is an experience based purely on
+what's in a computing device. AR on the other hand allows you to render virtual
+objects in the real world. To allow placement and tracking of those objects, we
+just added the
+[WebXR Hit Test API](https://github.com/immersive-web/hit-test/blob/master/explainer.md)
+to Chrome Canary, a new method that helps immersive web code place objects in
+the real world.
+
+<div class="video-wrapper">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="GDhujHhUEKc"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+## Where can I get it? {: #where }
+
+This API is intended to stay in Canary for the immediate future. We want a
+protracted testing period because this is a very new API proposal and we want to
+make sure it's both robust and right for developers.
+
+Aside from Chrome Canary, you'll also need:
+
+-  A
+   [compatible smartphone](/ar/discover/supported-devices) running
+   Android O or later.
+-  Two Chrome flags (chrome://flags): WebXRDevice API (#webxr) and WebXR Hit
+   Test (#webxr-hit-test)
+
+With these, you can dive into the demos or try out our codelab.
+
+Note: Some of the Immersive Web Community Group's
+[existing demos](https://immersive-web.github.io/webxr-samples/),
+specfically the ones using magic windows, do not work with the WebXR Hit Test
+turned on. Please excuse our construction debris.
+
+-  [Chacmool](https://web-education-ar-demo.appspot.com/)
+-  [Immersive Web](https://immersive-web.github.io/webxr-samples/proposals/phone-ar-hit-test.html)
+   Sample
+-  [Codelab](https://immersive-web.github.io/webxr-samples/phone-ar-hit-test.html)
+
+## It's just the web
+
+At Google IO this year, we demonstrated augmented reality with an early build of
+Chrome. I said something repeatedly to developers and non-developers alike
+during those three days that I wish I had known to put in my
+[immersive web article](/web/updates/2018/05/welcome-to-immersive):
+"It's just the web."
+
+<div class="video-wrapper">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="1t1gBVykneA"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+"What Chrome extension do I need to install?" "There's no extension. It's just
```

Comment:
I might change these to:
```
I had several questions at I/O that all had the same answer:

* What Chrome extension do I need to install?
* Do I need a special browser?
* What app do I need to install?

The answer to all was: **it's just the web**, it doesn't need any extensions
apps or anything other than your browser.
```

Reply:
I like it as dialog. Adds a little variety.

Reply:
Sure